tests/rbd/readonly_namespace_image_migration.py

```diff
@@ -0,0 +1,157 @@
+"""Module to execute Live migration feature on pool namespace.
+
+Pre-requisites :
+We need cluster configured with atleast one client node with ceph-common package,
+conf and keyring files
+
+Test cases covered :
+CEPH-83573341 - Set read only on namespace and migrate the images and
+verify if migration is successful
+
+Test Case Flow :
+1. Create pool and an image
+2. Create a namespace within the pool
+3. Give readonly access to the namespace
```

Comment:
Did we check whether read only namespace is a thing or it is just "read only client" to the namespace as I was mentioning in last week's discussion ?

Reply:
Hi @VasishtaShastry , there is no such setting to make namespace as read-only, we have to use cephx-caps like we discuss and also explore command to run with particular client id. 
We can use extra parameter as -n client.id  along with the exec_cmd.

Reply:
@rkhambad Can you please modify Doc and TC title to reflect that it is client which is read-only ?

Reply:
updated https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573341